app/controllers/users_controller.rb

```diff
@@ -24,11 +38,17 @@ def show
     @following_relationship = Relationship.new
   end
 
-  def update
-  end
-
   private
   def user_params
     params.require(:user).permit(:full_name, :password, :email)
   end
+
+  def handle_invitation
+    if params[:invitation_token].present?
+      invitation = Invitation.where(token: params[:invitation_token]).first
+      @user.follow(invitation.inviter)
+      invitation.inviter.follow(@user)
+      invitation.update_column(:token, nil)     
```

Comment:
It would be great to have this `.update_column(:token, nil)` part refactored to model under a name.